[API-8362]: govDelivery health check

Cgfor3 opened the pull request:
Split govDelivery health check into its own dedicated public route.
Update unit/integration tests to accommodate these changes.

Tests covered under [API-10487](https://vajira.max.gov/browse/API-10487)

TODO: Update tests so that code coverage requirement is met

Cgfor3 committed:
[API-8362]: govDelivery health check
Created dedicated public route for govDelivery health check

Cgfor3 committed:
Fix lint error

TS000 commented:
Looks like we need a little more test coverage in order to pass build checks:

`Jest: "/home/node/routes/HealthCheck.ts" coverage threshold for statements (80%) not met: 70%`

Cgfor3 committed:
Merge branch 'master' into API-8362-govdelivery-health-check

Cgfor3 committed:
- add GovDelivery health check unit test
- fix indentation issue
- TODO: Figure out why integration tests pass individually but fail running in a sequence
